torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -346,17 +346,17 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
     nodes = list(gm.graph.nodes)
     for node in nodes:
         g = GraphWrapper(node)
-        args = node.args
 
-        new_nodes = []
-        with g.inserting_before(node):
-            # TODO (kevinwuTT): consolidate and simplify these statements?
+        def rewrite_node(node):
```

Comment:
I remember in some cases this conversion created >1 node. Do you return a list?

Reply:
No, we only need the last node, the node that replaces the old node.  The current code uses only the last node as well.

https://github.com/tenstorrent/pytorch2.0_ttnn/blob/a5ce2a677f6ea9c6ba704c21db1c771bff03ef8a/torch_ttnn/passes/lowering/to_tt_pass.py#L592-L596

Reply:
For example, if the old graph is
```
Y = gemm(A, B, C)
```

And we convert it to a new graph
```
X = A @ B
Y = X + C
```

We only need to replace uses of the `gemm` node to the `add` node.  The intermediate `matmul` is good on its own (and Python takes care of its lifespan).